Src/Plugins/AzureDevOpsConfiguration/AZC101.BuildDefinitionSecurity.json

```diff
@@ -1,6 +1,8 @@
 {
-  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
   "ValidatorsAssemblyName": "AzureDevOpsConfiguration.dll",
+  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
+  "ExtensionName": "SecureAzureDevOps",
+  "Guid": "183f9c88-1424-4f6c-a8a5-cd59c0bdc312",
```

Comment:
>Guid [](http://example.com/codeflow?start=3&length=4)

we now declare a stable guid per extension though it currently isn't used. This will permit users to update the extension name, though, so we provide it so that consumers can use the stable guid rather than creating a potential for a breaking change by using the extension name as a key.